### PR TITLE
Check box delete all

### PR DESCRIPTION
**Pull request by Yosefgeda**
- Added functionality.
- completed tasks updated accordingly.

### CODE REVIEW

File: src/modules/todoClass.js [hunk 1]
```diff
@@ -67,4 +66,51 @@ export const editBtn = () => {
...
+  if (notDone.length === 0){
+    generatedList.style.display = 'none';
+  }
+  
+  const todoLists = document.querySelector('.todo-lists');
+
+  for (let i = 0; i < set.length; i++){
+    
+    if(set[i].completed === true){
+      //n = document.getElementById(`${set[i].id}`);
```
Comment (GedeonTS) [inline src/modules/todoClass.js]: - [ ] Kindly remove all commented out lines of code to make your code clean :recycle:

File: src/index.js [hunk 2]
```diff
@@ -43,16 +47,22 @@ addListBtn.addEventListener('click', (e) => {
...
     classObject.addToStorage(classObject.list);
   }
 });
 
+classObject.generatedList.addEventListener('click', classObject.deleteDone);
+clearAll.addEventListener('click', () => {
+  classObject.clear();
+  displayTodo();
+
+  // checkBox.parentElement.innerHTML = '';
```
Comment (GedeonTS) [inline src/index.js]: - Kindly remove all commented-out lines of code to make your code clean :recycle:

**Review by GedeonTS — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED :recycle: :

Hi @Yosefgeda  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights :tophat:  :+1: 
- [ ] The clear completed functionality is working fine :+1: 
- [ ] No linter errors :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mwanawabangona — APPROVED** [review]
Hi @Yosefgeda,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations!  


## Highlights
*   No linter errors :heavy_check_mark:
*   Repo looks professional :briefcase:
*   Solid pull request scroll
*   Good quality commits
*   Good job implementing the requested changes!! :clap: :clap: :clap:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**